deploy/scripts/src/spot/requestGenerator/spotReqGen.ts

```diff
@@ -0,0 +1,30 @@
+        'eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ1cm46ZmRjOmdvdi51azoyMDIyOnMzQkkwQjh3NHNvUEdnQm5aX3ktVXVyMWJKNWFYRUN2aUNram1XZThNR2ciLCJuYmYiOjE2NTIwMjAxMTAsImlzcyI6InBhc3Nwb3J0Y3JpIiwidm90IjoiUDIiLCJleHAiOjE2ODM2MjgxMTAsImlhdCI6MTY1MjA5MjExMCwidmMiOnsiZXZpZGVuY2UiOlt7ImFjdGl2aXR5SGlzdG9yeVNjb3JlIjowLCJ2YWxpZGl0eVNjb3JlIjoyLCJ2ZXJpZmljYXRpb25TY29yZSI6MCwic3RyZW5ndGhTY29yZSI6NCwidHhuIjoidHhuIiwiaWRlbnRpdHlGcmF1ZFNjb3JlIjowLCJ0eXBlIjoiSWRlbnRpdHlDaGVjayJ9XSwiY3JlZGVudGlhbFN1YmplY3QiOnsibmFtZSI6W3sibmFtZVBhcnRzIjpbeyJ0eXBlIjoiR2l2ZW5OYW1lIiwidmFsdWUiOiJKYW5lIn0seyJ0eXBlIjoiRmFtaWx5TmFtZSIsInZhbHVlIjoiV3JpZ2h0In1dLCJ2YWxpZEZyb20iOiIyMDE5LTA0LTAxIn0seyJ2YWxpZFVudGlsIjoiMjAyOS0wNC0wMSIsIm5hbWVQYXJ0cyI6W3sidHlwZSI6IkdpdmVuTmFtZSIsInZhbHVlIjoiSmFuZSJ9LHsidHlwZSI6IkZhbWlseU5hbWUiLCJ2YWx1ZSI6IldyaWdodCJ9XX1dLCJiaXJ0aERhdGUiOlt7InZhbHVlIjoiMTk4OS0wNy0wNiJ9XX0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJJZGVudGl0eUNoZWNrQ3JlZGVudGlhbCJdLCJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSIsImh0dHBzOi8vdm9jYWIubG9uZG9uLmNsb3VkYXBwcy5kaWdpdGFsL2NvbnRleHRzL2lkZW50aXR5LXYxLmpzb25sZCJdfSwidnRtIjoiaHR0cHM6Ly9vaWRjLmFjY291bnQuZ292LnVrL3RydXN0bWFyayJ9.MEQCIARWju22taR5B-kR6pnJnqW9NHNBotETORbHskk4MUkkAiAEWNtEr8dK25gZaxKjSyJJ-1Eikt1qSCzldHuSO5Xuyg' // pragma: allowlist secret
+      ],
+      vot: 'P2',
+      vtm: 'https://local.vtm'
+    },
+    in_local_account_id: 'a-simple-local-account-id',
+    in_rp_sector_id: 'a.simple.sector.id',
+    in_salt: 'YS1zaW1wbGUtc2FsdA==',
+    out_audience: uuidv4(),
+    log_ids: {
+      client_id: uuidv4(),
+      persistent_session_id: uuidv4(),
+      request_id: uuidv4() + '_' + currTime,
+      session_id: uuidv4(),
+      client_session_id: uuidv4()
+    },
+    out_sub: 'urn:fdc:gov.uk:2022:JG0RJI1pYbnanbvPs-j4j5-a-PFcmhry9Qu9NCEp5d4'
```

Comment:
Does this URN need to be parameterised?

Reply:
No this can be left hard coded. Only this URN will be used for the test and we have permission from SPOT Tech Lead that we can publish this data in a public repo.